semanticdb/metap/src/main/scala/scala/meta/internal/metap/SyntheticPrinter.scala

```diff
@@ -1,28 +1,84 @@
 package scala.meta.internal.metap
 
-import scala.math.Ordering
-import scala.meta.internal.semanticdb._
+import scala.meta.internal.{semanticdb => s}
```

Comment:
Other printers use the wildcard `import scala.meta.internal.semanticdb._`, so I suggest we do the same here for consistency.